sdk/storage/Azure.Storage.Common/src/Shared/ClientsideEncryption/GcmAuthenticatedCryptographicTransform.cs

```diff
@@ -34,7 +34,9 @@ public GcmAuthenticatedCryptographicTransform(byte[] key, TransformMode mode)
         {
             TransformMode = mode;
 
-#if NETSTANDARD2_1_OR_GREATER || NETCOREAPP3_1_OR_GREATER
+#if NET8_0_OR_GREATER
+            _gcm = new AesGcm(key, TagLength);
```

Comment:
See: [SYSLIB0053: AesGcm should indicate the required tag size](https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib0053)